### ISSUE TITLE
Correct Homepage that now even .listen('domain.tld') works

### ISSUE THREAD

**Issue by frank-dspeed** [adamhalasz/diet#23]
Correct Homepage that now even .listen('domain.tld') works

or Correct Homepage that now even .listen('domain.tld:222') works

**Comment by adamhalasz** [comment]
Do you mean this section?
http://dietjs.com/api/server#app.listen

**Comment by frank-dspeed** [comment]
yes there you can now add that it works with string that is not a url befor only http://domain.tld as string worked now domain.tld works without specify the protocol because the protocol now defaults to http if none supplyed and also a string without protocol with only domain name and port works like domain.tld:8080
